dbms/tests/queries/0_stateless/01071_in_array.sql

```diff
@@ -0,0 +1,8 @@
+select [1, 2] in [1, 2];
+select (1, 2) in (1, 2);
+select (1, 2) in [(1, 3), (1, 2)];
+select [1] in [[1], [2, 3]];
+select NULL in NULL;
+select ([1], [2]) in ([NULL], [NULL]);
+select ([1], [2]) in (([NULL], [NULL]), ([1], [2]));
+select ([1], [2]) in [([NULL], [NULL]), ([1], [2])];
```

Comment:
Would that make sense to  also check single elements (and single element-tuples/arrays)?:
```
select 1 in [1, 2];
select 3 in [1, 2];
select 1 in 1;
select 3 in 1;
select (1) in [1, 2];
select (3) in [1, 2];
select [1] in [1, 2];
select [3] in [1, 2];
```

Reply:
```
select [1] in [1, 2];
select [3] in [1, 2];
```

Not this case.

Reply:
Negative tests can be added...

Reply:
I've added this test, thank you!